Web/Admin.UI/src/app/components/tenant/global-operations/global-operations-search.component.html

```
@@ -1,269 +1,342 @@
+                        <div class="filter-item">
+                            <label>Facility</label>
+                            <select [(ngModel)]="facilityFilter">
+                                <option value="Any">Any</option>
+                                <option *ngFor="let option of facilityFilterOptions | keyvalue" [value]="option.key">{{ option.value }}
+                                </option>
+                            </select>
+                        </div>
+                        <div class="filter-item">
+                            <label>Vendor</label>
+                            <select [(ngModel)]="vendorFilter">
+                                <option value="Any">Any
```

HTMLHint (1.5.0)

[error] Tag must be paired, missing: [ </option> ], start tag match failed [ <option value="Any"> ] on line 97.

(tag-pair)